.github/workflows/deploy.yml

```diff
@@ -0,0 +1,95 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deploy
+
+on:
+  push:
+    branches: [ master, develop, cicd, '*staging*' ]
```

Comment:
Two things: 

1) We should rename 'master' to 'main'. 
2) We can delete the 'cicd' directive here once this branch has been merged.